src/main/java/se/sundsvall/billingpreprocessor/service/creator/ExternalInvoiceCreator.java

```diff
@@ -28,9 +30,12 @@
 
 import se.sundsvall.billingpreprocessor.integration.db.model.BillingRecordEntity;
 import se.sundsvall.billingpreprocessor.integration.db.model.InvoiceRowEntity;
+import se.sundsvall.billingpreprocessor.integration.db.model.enums.Type;
 
 @Component
-public class ExternalInvoiceCreator {
+public class ExternalInvoiceCreator implements InvoiceCreator {
+	private static final List<String> VALID_CAGTEGORIES = List.of("ISYCASE");
```

Comment:
Could this be read from DB instead?

(file_configuration contains valid categories, right?)